cwms_radar_api/build.gradle

```diff
@@ -237,6 +252,7 @@ task integrationTests(type: Test) {
     jvmArgs += "-Djava.util.logging.config.file=$projectDir/logging.properties"
     jvmArgs += "-Dorg.apache.tomcat.util.digester.PROPERTY_SOURCE=org.apache.tomcat.util.digester.EnvironmentPropertySource"
     jvmArgs += "-Dradar.access.provider=KeyAccessManager"
+    jvmArgs += "-Doracle.jdbc.Trace=true"
```

Comment:
Is this needed or left over?    I haven't played around with this -D, what does it do?  -Doracle.jdbc.Trace=true

Reply:
it was supposed to give more jdbc info to figure something out but it didn't work.